lambda_function.py

```diff
@@ -66,6 +76,29 @@ def generate_external_id() -> str:
 
         return str(uuid4())[:8]
 
+    def get_secret(self) -> Dict:
+        """
+        Get Dome9 AccessId and Secret for API authentication
+
+        :return:
+        """
+
+        try:
+            logger.info("Getting Dome9 API credentials from AWS SecretManager")
+            get_secret_value_response = self.secret_manager_client.get_secret_value(
```

Comment:
Please rename "get" sounds like an action - it's more appropriate for a function name